pkg/specgen/generate/container_create.go

```diff
@@ -166,7 +166,11 @@ func MakeContainer(ctx context.Context, rt *libpod.Runtime, s *specgen.SpecGener
 		options = append(options, libpod.WithName(s.Name))
 	}
 	if len(s.Devices) > 0 {
-		opts = ExtractCDIDevices(s)
+		registry := cdi.GetRegistry()
+		if errs := registry.GetErrors(); len(errs) > 0 {
+			logrus.Debugf("Error creating CDI Registry: %v", errs)
```

Comment:
Should we fail container creation here?

Reply:
No. These errors may not be related to the devices that we're interested in. For example, the spec from `vendorA` may be malformed, but we are using devices from `vendorB`.

Reply:
ok, makes sense